### ISSUE TITLE
Lexicon - Buttons lacking an aria-expanded attribute

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1593]
**Describe the bug**

There are several instances on the Lexicon where buttons with expandable/collapsible content are lacking an aria-expanded attribute. This means the state is not conveyed to screen reader users when activated which can lead to disorientation. Areas include:

1. At smaller viewports, the button toggle ‘Show Page Help’ which expands the additional aside slide out content lacks an aria-expanded attribute. 

2. At smaller viewports, under DataTable collapsible the button to expand or collapse the additional content lacks an aria-expanded attribute. 

**To Reproduce**
1. Go to Pulsar Lexicon
2. Reduce viewport width to under 627 px
3. Navigate to Tables > Datatables (collapsing)
4. Inspect 'Show page help' button
5. Inspect the toggle button within the table

**Expected behavior**
Aria-expanded attribute should be set on button elements that control the appearance of additional content to let assistive technology users know whether there are additional elements that are displayed and accessible or collapsed and hidden.

**Screenshots**

![Image](https://github.com/user-attachments/assets/dbb38fb9-6004-44d4-9840-7c9e23b8dcef)
![Image](https://github.com/user-attachments/assets/b0c0d4e2-eec4-4e05-834d-2686696f4bc8)

**Additional context**
Related success criteria:
[4.1.2 Name, Role, Value (A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Suggested fixes**

1. Add an aria-expanded attribute to the <button class="btn show-page-help js-show-page-help is-open"> and toggle value when opened/closed
2. Add an aria-expanded attribute to the <button class="table-child-toggle">
